sonar-project.properties

```diff
@@ -0,0 +1,10 @@
+sonar-project.properties
```

Comment:
I don't think this line is required, I haven't found it in the config file examples or in the configuration parameters documentation.

Reply:
I tried to run sonarqube without that line after SCoady suggested the same, and it wouldn't run so I left it in.